CONTRIBUTING.md

```diff
@@ -1,501 +1,108 @@
 # Contributing to Gno
 
-Thank you for looking to contribute to the Gno project.
-We appreciate every open-source contribution, as it helps us improve and enhance Gno for the benefit of the community.
-
-This document outlines some basic pointers on making your future contribution a great experience. It outlines basic PR
-etiquette employed by the core gno team. It lays out coding styles, simple how-to guides and tools to get you up and
-running and pushing code.
-
-If you are unsure about something, please don’t hesitate to reach out for help by opening an issue here or discuss on
-Discord.
-Likewise, if you have an idea on how to improve this guide, go for it as well.
-
-## Table of Contents
-
-- [Important Resources](#important-resources)
-- [Getting Started](#getting-started)
-    - [Environment](#environment)
-    - [Local Setup](#local-setup)
-    - [Testing](#testing)
-      - [Running locally](#running-locally)
-      - [Running test workflows](#running-test-workflows)
-      - [Testing Gno code](#testing-gno-code)
-    - [Repository Structure](#repository-structure)
-- [How do I?](#how-do-i)
-    - [How do I submit changes?](#how-do-i-submit-changes)
-      - [A Word on Rebasing](#a-word-on-rebasing)
-    - [How do I report a bug?](#how-do-i-report-a-bug)
-    - [How do I request a feature?](#how-do-i-request-a-feature)
-- [Style Guides](#style-guides)
-    - [Git Commit Messages](#git-commit-messages)
-    - [Go/Gno Style Guide](#gogno-style-guide)
-    - [Documentation Style Guide](#documentation-style-guide)
-- [Additional Notes](#additional-notes)
-    - [Issue and Pull Request Labels](#issue-and-pull-request-labels)
-      - [Labels for Pull Requests](#labels-for-pull-requests)
-      - [Labels for Issues](#labels-for-issues)
-    - [Notable Contributions](#notable-contributions)
-
-## Important Resources
-
-- **[Discord](https://discord.gg/YFtMjWwUN7)** - we are very active on Discord. Join today and start discussing all
-  things gno with fellow engineers and enthusiasts.
-- **[Awesome Gno](https://github.com/gnoverse/awesome-gno)** - check out the list of compiled resources for helping you
-  understand the gno ecosystem
-- **[Active Staging](https://staging.gno.land/)** - use the currently available staging environment to play around with a
-  production network. If you want to interact with a local instance, refer to the [Local Setup](#local-setup) guide.
-- **[Twitter](https://twitter.com/_gnoland)** - follow us on Twitter to get the latest scoop
-- **[Telegram](https://t.me/gnoland)** - join our official Telegram group to start a conversation about gno
+Thank you for contributing to Gno! This guide will help you get started.
 
 ## Getting Started
```

Comment:
I would keep the important resources above, maybe just linking docs, godoc, awesome gno and discord.